libs/execution_context/ebpf_native.c

```diff
@@ -592,7 +597,7 @@ _ebpf_native_validate_map(_In_ const ebpf_native_map_t* map, ebpf_handle_t origi
     ebpf_core_object_t* object;
     ebpf_handle_t inner_map_handle = ebpf_handle_invalid;
     uint16_t info_size = (uint16_t)sizeof(info);
-    ebpf_result_t result = ebpf_reference_object_by_handle(original_map_handle, EBPF_OBJECT_UNKNOWN, &object);
+    ebpf_result_t result = ebpf_object_reference_by_handle_and_type(original_map_handle, EBPF_OBJECT_UNKNOWN, &object);
```

Comment:
Shouldn't `EBPF_OBJECT_UNKNOWN` be `EBPF_OBJECT_MAP`?

Reply:
Changed to EBPF_OBJECT_MAP